Sources/Sentry/Public/SentryOptions.h

```diff
@@ -8,6 +8,12 @@ NS_ASSUME_NONNULL_BEGIN
 NS_SWIFT_NAME(Options)
 @interface SentryOptions : NSObject
 
+/**
+ * Optional, custom cache directory. Use when default one can not be accessed, e.g. in security
+ * environment.
+ */
+@property (nonatomic, copy) NSString *cacheDirectoryPath;
```

Comment:
`l`: Please move the property `parsedDsn`. It shouldn't be the first one in the Options file.

Reply:
I'm guessing you meant make `parsedDsn` the first one in the options file? I'll move `cacheDirectoryPath`.